src/compiler/python_generator.cc

```diff
@@ -236,6 +236,10 @@ bool PrintBetaServicer(const ServiceDescriptor* service,
              service->name());
   {
     IndentScope raii_class_indent(out);
+    out->Print("\"\"\"The Beta API is deprecated for 0.15.0 and later.\n"
+    "\nIt is recommended to use the GA API for all further purposes. This\n"
```

Comment:
"classes and functions in this file not marked beta" is a bit better. Take a look now.